uproot/source/compressed.py

```diff
@@ -130,10 +130,10 @@ def _prepare(self):
                 elif algo == b"L4":
                     compression = self.compression.copy(uproot.const.kLZ4)
                     compressedbytes -= 8
-                    checksum = cursor.bytes(self._compressed, 8)
+                    checksum = cursor.field(self._compressed, struct.Struct(">Q"))
```

Comment:
Good, though generally we predefine the `struct.Structs` (as `_fields0`, etc.).

You can probably reuse a lot of these techniques when writing LZ4 checksums.

Reply:
> Good, though generally we predefine the struct.Structs (as _fields0, etc.).

Should I make that change? 

> You can probably reuse a lot of these techniques when writing LZ4 checksums.

I don't think so because in the case of writing there are no conversions. We just have a bunch of bytes, put them into xxhash to get the checksum bytes and put them into the file.